### DISCUSSION TITLE
cannot use filter with item metadata on events

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3247] [Q&A]
Hello!
I have created an event "Weekend Workshops" with some products depending on the age group that these workshops refer to.
I would like to display multiple calendars depending on the age groups. So I created an item metadata on this event, named age_group and set for each product the corresponding age group.
I cannot filter them out in the widget.
For example the following pretix widget code
pretix-widget event="https://domain_name/organizer/weekend_workshops/" filter="attr[age_group]=2-4"
does not filter out the products for age group 2-4, but instead displays all products.

**Comment by Akhun-Delar** [comment]
Could be me, but did you forget to add your code?

**Reply by alemao8** [reply]
It was filtered because of the <>. I wrote it again without the special characters.

**Comment by wiffbi** [comment]
I am sorry, the meta-data filter in the widget only works on event level, not on item metadata.
